sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/Models/TagsOperations.cs

```diff
@@ -0,0 +1,432 @@
+        /// </summary>
+        /// <param name="parameters"> The tags to update. </param>
+        /// <param name="cancellationToken"> A token to allow the caller to cancel the call to the service. The default value is <see cref="CancellationToken.None" />. </param>
+        /// <returns> The updated tags. </returns>
+        /// <remarks>
+        /// <see href="https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-longrunning">Details on long running operation object.</see>
+        /// </remarks>
+        public virtual async Task<TagCreateOrUpdateOperation> StartUpdateAtScopeAsync(TagsPatchResource parameters, CancellationToken cancellationToken = default)
+        {
+            using var scope = Diagnostics.CreateScope("TagsOperations.StartUpdateAtScope");
+            scope.Start();
+            try
+            {
+                var response = await RestClient.UpdateAtScopeAsync(Id, parameters, cancellationToken).ConfigureAwait(false);
+                return new TagCreateOrUpdateOperation(this, response);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Get tags with the resource.
+        /// </summary>
+        /// <param name="cancellationToken"> A token to allow the caller to cancel the call to the service. The default value is <see cref="CancellationToken.None" />. </param>
+        /// <returns> The tags associate with resource. </returns>
+        /// <remarks>
+        /// <see href="https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-longrunning">Details on long running operation object.</see>
+        /// </remarks>
+        public virtual TagCreateOrUpdateOperation StartGetAtScope(CancellationToken cancellationToken = default)
+        {
+            using var scope = Diagnostics.CreateScope("TagsOperations.StartGetAtScope");
+            scope.Start();
+            try
+            {
+                var response = RestClient.GetAtScope(Id, cancellationToken);
+                return new TagCreateOrUpdateOperation(this, response);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Get tags with the resource.
+        /// </summary>
+        /// <param name="cancellationToken"> A token to allow the caller to cancel the call to the service. The default value is <see cref="CancellationToken.None" />. </param>
+        /// <returns> The tags associate with resource. </returns>
+        /// <remarks>
+        /// <see href="https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-longrunning">Details on long running operation object.</see>
+        /// </remarks>
+        public virtual async Task<TagCreateOrUpdateOperation> StartGetAtScopeAsync(CancellationToken cancellationToken = default)
```

Comment:
why do we need a StartGet?  we should never have Get which is an LRO

Reply:
Done